<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>citascalendarpac</class>
 <widget class="QWidget" name="citascalendarpac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1011</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_57">
        <item>
         <widget class="QLabel" name="lbl_paciente_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Nombre Medico:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_nombremed_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_61">
        <item>
         <widget class="QLabel" name="lbl_especialidad_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Especialidad:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_especialidad_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_30">
          <item>
           <widget class="QLabel" name="lbl_cita_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Hora de la Cita</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="time_horaCita_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QLabel" name="lbl_turno_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Turno:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_turno_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_58">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_33">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_34">
          <item>
           <widget class="QLabel" name="lbl_estado_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Estado:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_estado_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView_6">
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>350</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QCalendarWidget" name="calendarCitas_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>600</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="minimumDate">
         <date>
          <year>1990</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="maximumDate">
         <date>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Regresar al Menú</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
